scripts/execute_maestro_tests.sh

```diff
@@ -2,6 +2,8 @@
 
 reports_path="financial-connections-example/build/reports"
 logs_file_path="$reports_path/emulator.log"
+now=$(date +%F_%H-%M-%S)
+echo $now
```

Comment:
since there are several tries, this ensures recordings from different tries don't replace each other.